src/app/Http/Controllers/AdminController.php

```diff
@@ -1,10 +1,64 @@
 <?php
 
 namespace App\Http\Controllers;
+use App\Models\Order;
+use App\Models\Product;
+use App\Models\Categories;
+use App\Services\ManagerOrderService;
+use DB;
 
 class AdminController extends Controller
 {
-    public function index(){
-        return view('admin.index');
+    public function index(ManagerOrderService $managerOrderService)
+    {   
+        # get an array containing the number of orders for each month(lấy ra một mảng chứa số đơn đặt hàng của từng tháng).
+        $arr_Orders = Order::GetArrOrderMonth()->pluck('count_orders');
+        # get an array of months from 1 -> 12(lấy ra một mảng tháng từ 1 -> 12).
+        $arr_Months = Order::GetArrMonth()->pluck('month');
+        $number_of_orders = [0,0,0,0,0,0,0,0,0,0,0,0];
+
+        for ($i = 0; $i<count($arr_Months);$i++) {
+            $number_of_orders[$i] = $arr_Orders[$i];
+        }
```

Comment:
Mục đích đoạn này xử lý gì nhỉ?

Reply:
dùng for để lặp rồi add mảng $arr_Orders vừa select ở trên vào mảng $number_of_orders(mảng chứa dữ liệu mặc định cho trước),mục đích của việc này là ta sẽ tạo ra đc mảng chứa sl sản phẩm được đặt của từng sản phẩm đó trong vòng 1 năm